lib/storage.js

```diff
@@ -285,6 +285,7 @@ exports.getAll = function(options, callback) {
  * @param {Object} [options] - options
  * @param {String} [options.dataPath] - data path
  * @param {String} [options.validate] - validate writes by reading the data back
+ * @param {number} [options.spacing] - spacing passed to JSON.stringify
```

Comment:
Can we make it a `prettyprint` boolean option instead and default to 2 spaces. I don't think people are interested in the particular amount of spaces to use, but just in whether it is pretty printed or not.

Reply:
Sorry, I forgot to reply. I updated the pull request to replace it with a boolean option.